CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ Added
 -----
 
 - Added docs demonstrating how to apply an MSI patch with winrepo (#32780)
+- Added an execution and state module for running idem modules and states (#57969)
```

Comment:
This should just be a [changelog entry via towncrier](https://docs.saltstack.com/en/master/topics/development/changelog.html)